src/redux/store.js

```diff
@@ -0,0 +1,16 @@
+import { configureStore } from '@reduxjs/toolkit';
+import reserveItems from './reserveSlice';
+// import cartItems from './cartSlice';
+import tripSlice from './tripSlice';
+import sessionSlice from './reducers/sessionSlice';
+
+const store = configureStore({
+  reducer: {
+    reserveItems,
+    trips: tripSlice,
+    // cartItems,
```

Comment:
- [ ] Kindly, remove this all piece of commented code in order to improve your repo's cleanliness. 👍